content/typescript.md

```diff
@@ -2,7 +2,7 @@
 title: Typescript
 date: 2021-08-05T05:16:58+05:30
 category: language
-image: images/typescript.jpg
+image: https://upload.wikimedia.org/wikipedia/commons/4/4c/Typescript_logo_2020.svg
```

Comment:
hi @@riginoommen please upload the image to the cms using the file upload instead of using an external source.

Reply:
Also I guess the image there right now was created by the author(@anusham14) itself